src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -247,6 +256,13 @@ export class HuiDialogEditCard extends LitElement implements HassDialog {
                 </mwc-button>
               `
             : ``}
+          ${this._selectedEntities.length
+            ? html`
+                <mwc-button @click=${this._suggestCards}>
+                  ${this.hass!.localize("ui.common.continue")}
+                </mwc-button>
+              `
+            : ""}
```

Comment:
What happens when you select entities and then switch back to the by cards tab and select a card?